### PR TITLE
MODORDSTOR-31 receiving_history schema

### PR DESCRIPTION
**Pull request by javalkar-varun**
Define schema - https://issues.folio.org/browse/MODORDSTOR-31

### CODE REVIEW

File: mod-orders-storage/examples/receiving_history_collection.sample [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        "po_line_number": "268758-03",
+        "received_date": "2018-10-09T00:00:00.000Z",
+        "receiving_note": "ASD FGH ABCDEFGHIJKLMNOPQ",
+        "receiving_status": "Expected",
+        "supplement": true,
+        "title": "Kayak Fishing in the Northern Gulf Coast"
+      }
+    ],
+  "total_records": 1,
+  "first": 1,
```
Comment (craigmcnally) [inline mod-orders-storage/examples/receiving_history_collection.sample]: shouldn't first/last be 0 here?  or is do these fields really use 1-based indexing?

Comment (javalkar-varun) [reply mod-orders-storage/examples/receiving_history_collection.sample]: `total_records` `first` `last` start from 1. It is consistent with all other schemas we have till now.

Comment (craigmcnally) [reply mod-orders-storage/examples/receiving_history_collection.sample]: That seems confusing to me, but if that's really what the rest of FOLIO is doing I guess we should be consistent

Comment (craigmcnally) [reply mod-orders-storage/examples/receiving_history_collection.sample]: Now that I look I can't find any other places in FOLIO (besides acquisitions) that use this first/last.  This means, whatever precedent there is, it's our own, meaning we could change it if we wanted to.  The caveat there is that if we're going to do so we should soon, before other teams start integrating with our APIs.

@komick @bonthebeerman:  Do either of you know the history here?  Is this a carry-over from the JD days?  Is there a reason it's 1-based?

Comment (craigmcnally) [reply mod-orders-storage/examples/receiving_history_collection.sample]: Let's not let this hold us up... I'll create a user story to track this and hopefully we can find an answer and get some clarity on this.

Comment (craigmcnally) [reply mod-orders-storage/examples/receiving_history_collection.sample]: FYI:  https://issues.folio.org/browse/MODORDSTOR-37

**Pull request merged**